app/controllers/services_controller.rb

```diff
@@ -11,14 +11,66 @@ class ServicesController < ApplicationController
 
   def index
     if params[:category_id]
-      all_services = find_by_category(params[:category_id])
+      all_services = find_by_categories(params[:category_id], params[:eligibility_id])
     elsif params[:eligibility_id]
       all_services = find_by_eligibility(params[:eligibility_id])
     end
 
     render json: ServicesWithResourcePresenter.present(all_services)
   end
 
+  # rubocop:disable AbcSize
+  # rubocop:disable MethodLength
+  # rubocop:disable PerceivedComplexity
+  def find_by_categories(categories_id_string, eligibilities_id_string)
+    query_string = ""
+    if categories_id_string.include? ","
+      category_ids = categories_id_string.split ","
+      first_cat = true
+      category_ids.each do |cat_id|
+        if first_cat
+          query_string = "services.id in (select service_id from categories_services where category_id=" + cat_id + ")"
```

Comment:
This will expose us to a SQL injection attack. Can this be written using the ActiveRecord methods that properly escape external values? Or if you have to drop into raw SQL, at least use the sanitization methods available in ActiveRecord: https://api.rubyonrails.org/v5.1.0/

Reply:
Ooh that's a good point. I could just check if all of the incoming IDs are integers for example.